examples/openai/proxy.py

```diff
@@ -0,0 +1,11 @@
+from scrapegraphai.utils import search_proxy_servers
+
+proxies = search_proxy_servers(
+    anonymous=True,
+    countryset={"IT"},
+    # secure=True,
```

Comment:
I guess commenting out `secure=True` means none were being found.

that's what I'd document well: high security and low timeouts are inversely proportional